### PR TITLE
Fix #5905: AutoComplete: emptyMessageSelect listener

### LINKED ISSUE #5905: Autocomplete: Custom empty result action
## 1) Environment
- PrimeFaces version: 8.0

## 2) Expected behavior

Possibility to customize, container for no results found, so that it can for example add a button asking the user if he wants to create a new record, EX:

image

## 3) Actual behavior

..

## 4) Steps to reproduce

.. 

## 5) Sample XHTML

..

## 6) Sample bean

..

### PR DESCRIPTION
**Pull request by jepsar**
Fix https://github.com/primefaces/primefaces/issues/5905

### CODE REVIEW

**Review by melloware — APPROVED** [review]

File: src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js [hunk 1]
```diff
@@ -587,6 +587,18 @@ PrimeFaces.widget.AutoComplete = PrimeFaces.widget.BaseWidget.extend({
             $this.checkMatchedItem = false;
         });
 
+        this.panel.on('click', function() {
```
Comment (melloware) [inline src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js]: make it `click.emptyMessage` I like to add namespaces so when debugging we can easily tell where the onClick came from

**Review by melloware — CHANGES_REQUESTED** [review]

**Comment by tandraschko** [comment]
IMO "emptyMessage" as eventname is quite unclear when its triggered

**Comment by jepsar** [comment]
> IMO "emptyMessage" as eventname is quite unclear when its triggered

Agreed. I tried to keep it in line with `moreText`. I think both should be suffixed with "Click" or "Select".

**Comment by jepsar** [comment]
@tandraschko @melloware Shall I rename the events to `emptyMessageSelect` and `moreTextSelect` (and update the breaking changes section in the migration guide)?

**Comment by tandraschko** [comment]
i think so
WDYT @melloware ?

**Comment by melloware** [comment]
Yes I think we should its more specific name to what is happening.  I like "Select"

**Comment by jepsar** [comment]
Done (including migration guide).

**Review by melloware — APPROVED** [review]

**Pull request merged**